<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventanaOrdenacion</class>
 <widget class="QMainWindow" name="ventanaOrdenacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="boxTiempo1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiempo1">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>47</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo 1:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiempo2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>87</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo 2:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="boxTiempo2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiempo3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>127</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo3:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="boxTiempo3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>120</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiempo4">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>167</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo 4:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="boxTiempo4">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>160</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelTiempo5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>207</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo 5:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="boxTiempo5">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="botonEjecutar">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>247</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Ejecutar</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="boxMedia">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>294</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelMedia">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Media:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonResetear">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>341</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Resetear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botonVolver">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Volver</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>408</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
